subworkflows/local/annotate.nf

```diff
@@ -93,25 +93,36 @@ workflow ANNOTATE {
     }
 
     CHECKV(
-      predicted_contigs.combine( contigs.map { name, fasta -> fasta }),
+      predicted_contigs.join(contigs.map { meta, fasta -> fasta }),
       checkv_db
     )
 
-    viphos_annotations = ANNOTATION.out.map { _, __, annotations -> annotations }.collect()
-    taxonomy_annotations = ASSIGN.out.map { _, __, taxonomy -> taxonomy }.collect()
-    checkv_results = CHECKV.out.map { _, __, quality_summary, ___ -> quality_summary }.collect()
+    viphos_annotations = ANNOTATION.out.annotations.groupTuple().map{ 
```

Comment:
Is this `groupTuple` required becase `ANNOTATION` return a list of files? I'm not sure I follow

Reply:
it is in development stage yet, I'm not sure what it returns either :D

Reply:
Perfect hehe